scripts/deployment20/test-configs/opt-mainnet.conf.ts

```diff
@@ -88,6 +90,14 @@ export const config = {
           'address:ServiceRegistry',
         ],
       },
+      ChainLogView: {
```

Comment:
Let's rename the whole file to `optimism.conf` again. Also we don't need that/ You don't maker's `chainlog` on optimism. Also no idea where you go that  address from ( the one for the constructor )